rust/repositories.bzl

```diff
@@ -597,7 +608,9 @@ def rust_repository_set(
             sha256s = sha256s,
             urls = urls,
             auth = auth,
-        ))
+        )
+
+        all_toolchain_names.append("@{name}//:toolchain".format(name = name))
 
     # This repository exists to allow `rust_repository_set` to work with the `maybe` wrapper.
```

Comment:
I can't say that I really understand the point of this repository. `ALL_TOOLCHAINS` from the repository does not seem to be used anywhere. Can anyone explain this to me?

Reply:
To my knowledge, the content of this repository is not used. As the comment states, it's only there to allow `rust_repository_set` to satisfy the `maybe` interface.

Reply:
Makes sense.